tests/utils.go

```diff
@@ -1120,10 +1118,14 @@ func createSeparateDeviceHostPathPv(osName, nodeName string) {
 }
 
 func CreateHostPathPv(osName string, hostPath string) string {
-	return CreateHostPathPvWithSize(osName, hostPath, "1Gi")
+	return createHostPathPvWithSize(osName, hostPath, "1Gi")
 }
 
-func CreateHostPathPvWithSize(osName string, hostPath string, size string) string {
+func createHostPathPvWithSize(osName string, hostPath string, size string) string {
+	return CreateHostPathPvWithSizeAndStorageClass(osName, hostPath, size, Config.StorageClassLocal)
+}
+
+func CreateHostPathPvWithSizeAndStorageClass(osName string, hostPath string, size string, sc string) string {
```

Comment:
In go if you have multiple parameters with a single type you can specify the type only one like this:
```go
func CreateHostPathPvWithSizeAndStorageClass(osName, hostPath, size, sc string) string {
...
}
```